chain-impl-mockchain/tests/common/arbitrary/transaction.rs

```diff
@@ -0,0 +1,45 @@
+use crate::common::address::AddressData;
+use chain_addr::Address;
+use chain_impl_mockchain::transaction::Output;
+use chain_impl_mockchain::value::Value;
+use quickcheck::{Arbitrary, Gen};
+
+#[derive(Clone, Debug)]
+pub struct ArbitraryValue(pub Value);
+
+impl Arbitrary for ArbitraryValue {
```

Comment:
This could be defined for `Value` itself in the mockchain crate, under `mod testing` (does not have to be public) conditionally enabled on `cfg(feature = "property-test-api")` like in the other crates.